scripts/spack/packages/serac/package.py

```diff
@@ -139,6 +142,9 @@ class Serac(CachedCMakePackage, CudaPackage):
 
     depends_on("superlu-dist@8.1.2")
 
+    # The optional slate dependency is not handled in the MFEM spack package
+    depends_on("strumpack~slate~butterflypack~shared", when="+strumpack")
```

Comment:
You should remove the `~shared` and add `strumpack` to the for loop on line 150.